search-scores-and-weights.md

```diff
@@ -16,38 +15,42 @@ and weights to documents based on various parameters, influencing their ranking
+   reflecting the relevance of the document to the search criteria. It is not a
+   static precomputed score but a dynamic metric that is computed on the fly
+   from the search results. This ensures that the relevance is tied to the
    specific query, going beyond simple keyword matching. Documents with higher
    similarity scores are considered more relevant. By prioritizing similarity as
    the first signal in our scoring process, we aim to deliver search results
    that are more accurate.
 
-1. **Recency**: This score considers the temporal aspect of documents,
-   prioritizing recently added or updated content. A document's recency is
-   crucial in reflecting the latest information available to users.
+1. [**Recency**](sql/schema2.sql.public_new.sql): This score considers the
+   temporal aspect of documents, prioritizing recently added or updated content.
+   A document's recency is crucial in reflecting the latest information
+   available to users.
 
-1. **Traffic**: The frequency with which users consult a document influences its
-   score. Popular or frequently accessed documents are given higher scores with
-   the help of web traffic logs, indicating their relevance and importance to
-   users. Warning: The home page is rated really high since it's where every
-   user land at first.
+1. [**Traffic**](sql/compute-traffic-score.sql): The frequency with which users
+   consult a document influences its score. Popular or frequently accessed
+   documents are given higher scores with the help of web traffic logs,
+   indicating their relevance and importance to users. Warning: The home page is
+   rated really high since it's where every user land at first.
 
-1. **Current**: This score determines whether a document is currently accessible
-   (current=1) or if it has been archived (current=0). It helps users
-   distinguish between active and inactive content.
+1. [**Current**](sql/2023-07-12-score-current.sql): This score determines
+   whether a document is currently accessible (current=1) or if it has been
+   archived (current=0). It helps users distinguish between active and inactive
+   content.
 
-1. **Typicality**: This score evaluates how closely the number of site
-   references for a document aligns with the average within a given thematic
-   context. Documents with typicality scores reflect a level of correspondence
-   with the average number of references, indicating their alignment with the
-   common standards within the specified topic. This ensures that the search
-   results prioritize documents not only based on their relevance to the user's
-   query but also considering how well they conform to the typical reference
-   patterns within the targeted theme.
+1. [**Typicality**](sql/2023-07-12-calculate-incoming-outgoing-counts.sql): This
+   score evaluates how closely the number of site references for a document
+   aligns with the average within a given thematic context. Documents with
```

Comment:
what is a thematic context?

Reply:
Thematic context means a context concerning the subject or theme of something, it's like a specific context given the user's query

Reply:
Oh, so maybe I misunderstood what this score is then, I'll look into the SQL more carefully